cumulus_library_opioid/vocab/rxnorm_vsac_builder.py

```diff
@@ -34,12 +34,11 @@ def get_create_view_filter_by(
         ):
             a_schema = a_schema or 'rxnorm.'
             a_join_col = a_join_col or 'a.rxcui'
-            b_join_col = b_join_col or 'b.rxcui'
-            b_table = b_table or f'opioid__{steward}_vsac',
+            b_join_col = b_join_col or 'b.code'
+            b_table = b_table or f'opioid__{steward}_vsac'
```

Comment:
Python is wrong here :smile: - it should really require parens or something to make the theoretical "I want a tuple" clearer

Reply:
this has bitten me in both directions more times than i'd care to count